spec/factories/in_progress_forms/in_progress_forms.rb

```diff
@@ -7,7 +7,7 @@
     metadata do
       {
         version: 1,
-        return_url: 'foo.com'
```

Comment:
Should this be changed?  most forms will still be sending the header, so more often this will be snake case

Reply:
yeah, i was conflicted for that reason. i'm fine with swapping it back 👍